src/float.rs

```diff
@@ -1917,6 +1946,11 @@ macro_rules! float_impl_std {
                 Self::acosh(self) -> Self;
                 Self::atanh(self) -> Self;
             }
+
+            #[cfg(has_copysign)]
+            forward! {
+                Self::copysign(self, sign: Self) -> Self;
+            }
```

Comment:
Hmm, 1.8.0 seems to expand this even if not `cfg(has_copysign)`. While it correctly leaves it out if the function is written directly instead of using the `forward!` macro. That is it correctly leaves out
```rust
            #[cfg(has_copysign)]
            #[inline]
            fn copysign(self, sign: Self) -> Self {
                Self::copysign(self, sign)
            }
```

After some investigation, it seems to me that this bug does not affect a `#[cfg(...)] forward!{...}` that is not inside another macro, so this is the only place I could find in the file. So simply expanding this single instance of the `forward!` macro should fix the build I think.

Reply:
Thanks for the investigation, 1.8.0 seems to just not work on my MacBook so I wasn't able to test 1.8.0. I've updated the code with your suggestion.